training_system/app/models/user.rb

```diff
@@ -8,7 +8,8 @@ class User < ApplicationRecord
   validates :email, presence: true, length: {maximum: Settings.email_maxlen},
                     format: {with: VALID_EMAIL_REGEX},
                     uniqueness: {case_sensitive: false}
-  validates :password, presence: true, length: {minimum: Settings.pass_minlen}
+  validates :password, presence: true, length:
+            {minimum: Settings.pass_minlen}, allow_nil: true
```

Comment:
ko nên break giữa chừng xử lý nha, xuống dòng từ sau dấu phẩy nha
```suggestion
  validates :password, presence: true,
            length: {minimum: Settings.pass_minlen}, allow_nil: true
```

Reply:
fixed